index.js

```diff
@@ -0,0 +1,229 @@
+// Resume Builder
+const fs = require('fs');
+const readline = require('readline');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout
+});
+
+let resume = {
+    "basics": {
+        "name": "",
+        "label": "",
+        "picture": "",
+        "email": "",
+        "phone": "",
+        "website": "",
+        "summary": ""
+      },
+      "work": [],
+      "education": [],
+      "skills": [],
+};
```

Comment:
Building out the resume JSON skeleton seems like a good idea, something I think I should have done in retrospect.